trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/add-to-an-existing-payment-helper.ts

```diff
@@ -0,0 +1,28 @@
+import { SelectedFeeRecordsAvailablePaymentGroups } from '@ukef/dtfs2-common';
+
+export const getAvailablePaymentsHeading = (paymentGroups: SelectedFeeRecordsAvailablePaymentGroups) => {
+  if (!paymentGroups || paymentGroups.length === 0) {
+    throw new Error('No payment groups to select from.');
+  }
+
+  const numOfSinglePayments = paymentGroups.filter((group) => group.length === 1).length;
+  const numOfGroupPayments = paymentGroups.filter((group) => group.length > 1).length;
+
+  if (paymentGroups.length === 1) {
+    if (numOfSinglePayments === 1) {
+      return 'There is one existing payment that the reported fees will be added to';
+    }
+
+    return 'There is one existing group of payments that the reported fees will be added to';
+  }
+
+  if (numOfSinglePayments > 1 && numOfGroupPayments === 0) {
```

Comment:
I think this could be simplified to just `numOfGroupPayments === 0`

Reply:
Or actually might be nicer to have `numOfSinglePayments === totalNumberOfPaymentGroups` since that is perhaps what are brains are needing to understand from this i.e. all payments are single payments (pulling out a variable for the total for readability)